static/sass/styles.scss

```diff
@@ -1378,6 +1378,14 @@ $color-link-dark: #69c !default;
   }
 }
 
+.p-logo-section--dense {
+  .p-logo-section__items {
+    .p-logo-section__item > img {
+      height: 96px;
+    }
+  }
+}
+
```

Comment:
We shouldn't be adding styles directly changing Vanilla classes, it can modify other images unintendedly